src/mscorlib/src/System/Threading/ThreadPool.cs

```diff
@@ -145,7 +143,7 @@ internal class WorkStealingQueue
             private volatile int m_headIndex = START_INDEX;
             private volatile int m_tailIndex = START_INDEX;
 
-            private SpinLock m_foreignLock = new SpinLock(false);
+            private readonly SpinLock m_foreignLock = new SpinLock(false);
```

Comment:
Thanks, fixed.

I wonder if there some reasonable way to automatically catch such errors in the future.